ex5/server/db/migrations/20220618150319-create-items.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+module.exports = {
+	async up(queryInterface, Sequelize) {
```

Comment:
you went back and defined the status column in the previous migration.
when I ran the migrations it errored out because status was already defined..
I had to comment this line out for the project to run

Reply:
It's best practice to not change migration files after you've ran them 
You could very easily create conflicts for other developers on your project